portfolio/src/main/webapp/style.css

```diff
@@ -9,9 +9,21 @@ button {
   font-family: 'Courier New', Courier, monospace;
 }
 
+.comment {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  padding: 10px;
+}
+
+.comment img {
+  border-radius: 50%;
+  width: 6%;
+}
+
 #comments-container {
```

Comment:
As a stretch goal, consider making sure this works well with different window sizes.